packages/@react-spectrum/dialog/src/Dialog.tsx

```diff
@@ -71,7 +71,7 @@ function Dialog(props: SpectrumDialogProps, ref: DOMRef) {
     content: {UNSAFE_className: styles['spectrum-Dialog-content']},
     footer: {UNSAFE_className: styles['spectrum-Dialog-footer']},
     buttonGroup: {UNSAFE_className: classNames(styles, 'spectrum-Dialog-buttonGroup', {'spectrum-Dialog-buttonGroup--noFooter': !hasFooter}), align: 'end'}
-  }), [styles, hasFooter, hasHeader, titleProps]);
+  }), [hasFooter, hasHeader, titleProps]);
```

Comment:
more lint cleanup? styles should just be the same object from initial loading of the module, or is it changing sometimes?

Reply:
This was a lint cleanup. Should actually be:
`[hasFooter, hasHeader, hasHeading, hasTypeIcon, titleProps]`.

Reply:
`styles` isn't changing. It contains the classNames from spectrum.